test/app/components/collection_parameter_with_active_model_component.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class CollectionParameterWithActiveModelComponent < ViewComponent::Base
+  include ActiveModel::Model
+  include ActiveModel::Attributes
```

Comment:
Do we need to include both of these?

Reply:
Most likely not. I'll play Jenga with this.